lte/protos/session_manager.proto

```diff
@@ -36,20 +36,54 @@ enum RATType {
 }
 
 message LocalCreateSessionRequest {
+  SubscriberID sid = 1; // Included in Common ctx
+  string ue_ipv4 = 2; // Included in Common ctx
+  string spgw_ipv4 = 3; // Included in LTE ctx
+  string apn = 4; // Included in Common ctx
+  string imei = 6; // Included in LTE ctx
+  string plmn_id = 7; // Included in LTE ctx
+  string imsi_plmn_id = 8; // Included in LTE ctx
+  bytes user_location = 9; // Included in LTE ctx
+  QosInformationRequest qos_info = 10; // Included in LTE ctx
+  bytes msisdn = 11; // Included in Common ctx
+  RATType rat_type = 12; // Included in Common ctx
+  bytes hardware_addr = 13; // MAC Address for WLAN
+  string radius_session_id = 14; // Included in WLAN ctx
+  uint32 bearer_id = 15; // Included in LTE ctx
+  CommonSessionContext common_context = 16;
+  RatSpecificContext rat_specific_context = 17;
+}
+
+message RatSpecificContext {
```

Comment:
what about moving this definition after `commonSessionContext`, this way it will be just before the contexts it encapsulate

Reply:
makes sense